runTests.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+PS_VERSION="${PS_VERSION:-1.7}"
+URL="${URL:-prestashop}"
+MODULE="${MODULE:-}"
+SAUCELABS="${SAUCELABS:-}"
+SCRIPT="${SCRIPT:-index.webdriverio.js}"
+
+COMMAND="/tmp/node_modules/mocha/bin/mocha /tmp/test/itg/$PS_VERSION/$SCRIPT --URL=$URL"
+
+if [ -n "$MODULE" ]; then
+    COMMAND="$COMMAND --MODULE=$MODULE"
```

Comment:
To be checked with @basma-yangui-prestashop. If I remember, we always need to set a value here, `none` if we do not test any module.

We may need to update the JS code as well?